scripts/variantstore/README.md

```diff
@@ -1,9 +1,31 @@
-
-
+# Running the Genome Variant Store (GVS) Pipeline for AoU
+
+## Steps
+1. If this is the first time running the GVS pipeline in a particular Google billing project, use your GCP account team to create a support ticket for the BigQuery team that includes "enabling cluster metadata pruning support for the BQ Read API."
+2. This list assumes you are starting off with a Terra workspace that contains the re-blocked gVCFs for your callset as "sample sets" in the "Data" tab.  For workflows, you will be following the steps outlined in the [GVS Quickstart workspace](https://app.terra.bio/#workspaces/broad-dsde-firecloud-billing/Genomic%20Variant%20Store%20-%20GVS%20Quickstart).  Make special note of the "Prerequisites" section.
+3. Run the workflows outlined in the Quickstart description in the workspace that contains the data by using the "Copy to Another Workspace" function.  The first two steps (GvsAssignIds and GvsImportGenomes) can be run multiple times to load all necessary sample sets into one "instance" of GVS.  The rest of the steps are only run once, in order, and only after all the samples for the callset are loaded.
+   1. GvsAssignIds at the sample set level ("Step 2" in workflow submission)
+   2. GvsImportGenomes at the sample set level ("Step 2" in workflow submission)
```

Comment:
sorry to be picky. but it looks like it's step 1 where you set the entity type to sample_set

Reply:
I thought the entity type was sample, but then in Step 2 you select sample sets to run them on?  I could have this wrong, tho...